package.json

```diff
@@ -48,7 +48,7 @@
   "scripts": {
     "prepare": "yarn prepare:travis && yarn prepare:hoisting && yarn rebuild:clean && yarn build:clean",
     "prepare:travis": "node scripts/prepare-travis",
-    "prepare:hoisting": "node scripts/check-hoisting",
+    "prepare:hoisting": "ts-node scripts/check-hoisting.ts",
```

Comment:
we are planning to ditch ts-node, please use js

Reply:
Actually, I was thinking that this script should be used by app developers to check the consistency of the final app. In this case it should be a part of `application-manager` package and there you can use TS.

@kittaakos does it make sense? 

It could be a separate PR for sure.

Reply:
May I inquire why?  Using Typescript here clearly shows that it can avoid bugs.

Reply:
I am OK with moving to the `application-manager`.

Reply:
> Actually, I was thinking that this script should be used by app developers to check the consistency of the final app. In this case it should be a part of `application-manager` package and there you can use TS.
> 
> @kittaakos does it make sense?
> 
> It could be a separate PR for sure.

IIUC, that would allow us to use it in applications built outside the main Theia repo?  That would be nice, we were actually wondering about that.

Reply:
I propose to use `ts-node` for now, until we move the script to `application-manager`.

Reply:
@simark https://github.com/theia-ide/theia/pull/2563 because it is slow

Reply:
I agree that for the tests it would be much better not to use ts-node.  Since the tests are compiled during the build anyway, we can execute the already compiled js directly.

But for a small one-off script like this, where using Typescript clearly has advantages, I think ts-node is fine.  Anyway, moving it to application-manager will make this irrelevant as it will be compiled with the rest.